src/templates/static/vulnerabilities/cves.yml

```diff
@@ -2,7 +2,28 @@
   hosts: "@@HOSTS@@"
   vars:
     insights_issues: "@@ISSUES@@"
-    insights_signature_exclude: "/hosts,/vars/insights_issues"
+    insights_signature_exclude: /hosts,/vars/insights_signature,/vars/insights_issues
+    insights_signature: !!binary |
```

Comment:
is `!!binary` needed for the signature?

Reply:
@jharting I believe so yes.  Because of us having to support python 3.6 & 2.7 the library we use for dumping / encoding the signature makes it default to this format.